coil-sample/src/main/assets/images.json

```diff
@@ -1,15 +1,39 @@
 [
     {
         "urls": {
-            "raw": "https://images.unsplash.com/photo-1550973886-796d048c599f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjU4MjM5fQ",
-            "full": "https://images.unsplash.com/photo-1550973886-796d048c599f?ixlib=rb-1.2.1&q=85&fm=jpg&crop=entropy&cs=srgb&ixid=eyJhcHBfaWQiOjU4MjM5fQ",
-            "regular": "https://images.unsplash.com/photo-1550973886-796d048c599f?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=1080&fit=max&ixid=eyJhcHBfaWQiOjU4MjM5fQ",
-            "small": "https://images.unsplash.com/photo-1550973886-796d048c599f?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=400&fit=max&ixid=eyJhcHBfaWQiOjU4MjM5fQ",
-            "thumb": "https://images.unsplash.com/photo-1550973886-796d048c599f?ixlib=rb-1.2.1&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=200&fit=max&ixid=eyJhcHBfaWQiOjU4MjM5fQ"
+            "raw": "https://cdn.coinranking.com/Sy33Krudb/btc.svg",
```

Comment:
I figured it made sense to include these in the sample app, so it details how to use these dependencies, but I'd assume the final sample should use different imagery.  AFAIK Unsplash doesn't offer SVGs, so I'm not really sure where to get them from.

Reply:
Let's use new SVGs please, not sure if we're allowed to use their assets in this way. Found a few samples that could work:

https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/android.svg
https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/mac.svg
https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/photos.svg

Reply:
@FrostRocket See: https://github.com/coil-kt/coil/pull/38#discussion_r314913499